h/static/scripts/annotator/guest.coffee

```diff
@@ -197,10 +209,14 @@ module.exports = class Guest extends Annotator
   _setupViewer: -> this
   _setupEditor: -> this
   _setupDocumentEvents: ->
-    $(document).bind({
```

Comment:
If you bind the events together, it breaks being able to create an annotation in iOS Safari. It also leads to a weird experience on Android where the Adder sometimes shows but sometimes doesn't.